data_steward/analytics/cdr_ops/rdr_export_qc.py

```diff
@@ -26,40 +27,45 @@
 import urllib
 import pandas as pd
 
+from google.oauth2 import service_account
+
 pd.options.display.max_rows = 120
+
+creds = service_account.Credentials.from_service_account_file(key_file)
```

Comment:
Could we avoid handling credentials and just fall back on application default credentials as we do with other scripts?

Reply:
I added this because I had not run `gcloud auth application-default login` before and did not have any default credentials.  Running from the command line, the script just failed.  It didn't open a browser for me to authenticate with.  I will remove this for now, but I'd like to discuss setting this up to run with impersonated credentials/user provided credentials in the future.

Reply:
What am I missing?  Without explicitly creating the credentials, I get the following error when trying to run the notebook.
```
     10 WHERE curr.table_id IS NULL OR prev.table_id IS NULL
     11 '''
---> 12 pd.read_gbq(query, dialect='standard')

... stack trace shortened for readability

/Users/lbiggers/Projects/env37/lib/python3.7/site-packages/ipykernel/kernelbase.py in raw_input(self, prompt)
    697         if not self._allow_stdin:
    698             raise StdinNotImplementedError(
--> 699                 "raw_input was called, but this frontend does not support input requests."
    700             )
    701         return self._input_request(py3compat.cast_unicode_py2(prompt),

StdinNotImplementedError: raw_input was called, but this frontend does not support input requests.
```

Reply:
I was able to use this line to run the notebook.

```
GOOGLE_APPLICATION_CREDENTIALS=~/.keys/XXXXXXXX.json PYTHONPATH=./:$PYTHONPATH  python analytics/cdr_ops/report_runner.py analytics/cdr_ops/rdr_export_qc.py -p project_id <project_value> -p old_rdr <old_value> -p new_rdr <rdr_value>
```
I just really dislike assuming someone has set their path correctly.  This goes back to requiring someone to explicitly choose between prod and test.  It's so easy to set the wrong variable and move forward altering the wrong data.

But, this is a conversation to have outside of this PR.

Reply:
I would definitely like to discuss how authentication should work in different use cases and then make everything work consistently. Let's just tackle that as a separate issue please :pray:  because there are docs and things that are affected by such changes.

Reply:
I agree it is too large of an issue to tackle in this PR.  So, I've already removed that code.